src/iqm/benchmarks/benchmark_definition.py

```diff
@@ -34,6 +34,52 @@
 from iqm.qiskit_iqm.iqm_provider import IQMBackend, IQMFacadeBackend
 
 
+@dataclass
+class ObservationIdentifier:
+    """Identifier for observations for ease of use
+
+    Attributes:
+        qubit_indices: list containing the indices of the qubits the observation was executed on.
+    """
+
+    qubit_indices: list[int]
+
+    @property
+    def string_identifier(self) -> str:
+        """String version of the qubit indices for ease of use
```

Comment:
Is this supposed to be a single string like `'[1,2,3]'`, or a list of string representations like `['1', '2', '3']`?

Reply:
A single string at this point. Honestly not strictly needed, but I was hoping the identifier would evolve in the future since referencing, for example, 1k qubits by their indices does not seem to be very nice or useful...

Reply:
What is the use case for this identifier? Is it meant to be used in some persistent storage as a unique identifier of the observation?